packages/babel-plugin/src/utils/ast-builders.tsx

```diff
@@ -68,15 +89,30 @@ const buildCssVariablesProp = (
  */
 const styledStyleProp = (
   variables: CSSOutput['variables'],
-  transform?: (expression: t.Expression) => any
+  transform?: (expression: t.Expression) => t.Expression
 ) => {
   const props: (t.ObjectProperty | t.SpreadElement)[] = [t.spreadElement(t.identifier('style'))];
   return t.objectExpression(props.concat(buildCssVariablesProp(variables, transform)));
 };
 
-const buildComponentTag = ({ name, type }: Tag) =>
-  type === 'InBuiltComponent' ? `"${name}"` : name;
+/**
+ * Returns a tag string in the form of an identifier or string literal.
+ *
+ * A type of InBuiltComponent will return a string literal,
+ * otherwise an identifier string will be returned.
+ *
+ * @param param0
```

Comment:
nit: Should we add description for parameters here?

Reply:
probably 🤔 😅